src/web/views/guidance/en/using-your-card.njk

```diff
@@ -0,0 +1,79 @@
+
+{% block guidanceContent %}
+<h2 class="govuk-heading-l">Using your card</h2>
+<p class="govuk-body">You’ll get a card in the post with instructions on how to activate and use it. It will be sent to you for free.</p>
+<p class="govuk-body">You’ll be given a 4-digit PIN to use when making payments and to check your balance. You will be able to change this so it’s easy to remember.</p>
+<p class="govuk-body">Once activated you can use your card straight away.</p>
+
+<h3 class="govuk-heading-m">Contactless payments</h3>
+
+<p class="govuk-body">The first time you make a payment you’ll have to use your PIN. After that you can make contactless payments up to £30.</p>
+<p class="govuk-body">You’ll sometimes be asked to enter your PIN for contactless payments – this is for extra security.</p>
+<p class="govuk-body">You’ll have to enter your PIN for payments over £30.</p>
+
+<h3 class="govuk-heading-m">Checking your balance</h3>
+
+<p class="govuk-body">You can check your balance at any cashpoint that allows this. </p>
+<p class="govuk-body">You cannot withdraw cash.</p>
+
+<hr class="govuk-section-break govuk-section-break--l">
+<h2 class="govuk-heading-l">Where you can shop</h2>
+
+<p class="govuk-body">You can use your card in most places that sell milk, infant formula, fruit and vegetables. These include:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>corner shops</li>
+  <li>supermarkets</li>
+  <li>food co-ops</li>
+  <li>sweet shops</li>
+  <li>grocery shops including discount stores</li>
+  <li>greengrocers</li>
+  <li>market stalls</li>
+  <li>milk floats</li>
+  <li>food markets</li>
+  <li>petrol stations</li>
+  <li>pharmacies</li>
+  <li>newsagents and newsstands</li>
+  <li>hospital grocery shops</li>
+  <li>bakeries</li>
+</ul>
+<p class="govuk-body">They must display a Mastercard&#174; symbol.</p>
+
+<div class="govuk-inset-text">If you try to spend the money anywhere else the card will be declined.</div>
+
+<hr class="govuk-section-break govuk-section-break--l">
+<h2 class="govuk-heading-l">If your situation changes </h2>
+
+<p class="govuk-body">If there’s any money left on the card when your entitlement ends you have 16 weeks to spend it.</p>
+<p class="govuk-body">If you do not spend it in this time you will not get a refund.</p>
+<p class="govuk-body">The same applies if you cancel the card yourself.</p>
+
+<hr class="govuk-section-break govuk-section-break--l">
+<h2 class="govuk-heading-l">Reporting a card problem</h2>
+
+<h3 class="govuk-heading-m">Reporting activation problems</h3>
+
+<p class="govuk-body">If you experience any problems activating or using your card:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>call 0300 330 7010</li>
+  <li>select option 2 and follow the instructions</li>
+</ul>
+
+<h3 class="govuk-heading-m">Lost or stolen cards</h3>
+
+<div class="govuk-inset-text">You are responsible for all spending on your prepaid card even if someone else uses it.</div>
```

Comment:
I think there is a Nunjucks macro for this https://design-system.service.gov.uk/components/inset-text/

Reply:
Done.